build/cmake/package_tester/modules/tests.sh

```diff
@@ -59,6 +59,14 @@ then
        then
           fail "value was not cleared correctly"
        fi
+       PYTHON_TARGZ_NAME="$(ls /build/packages | grep 'foundationdb-[0-9.]*\.tar\.gz' | sed 's/\.tar\.gz$//')"
+       tar -C /tmp -xvf /build/packages/${PYTHON_TARGZ_NAME}.tar.gz
+       pushd /tmp/${PYTHON_TARGZ_NAME}
+       python setup.py install
```

Comment:
I assume you verified that all docker-images have python installed?

Reply:
The debian package does not, but if we got to this point without python the test would have already failed on debian, since apparently the server deb depends on python being installed:

```dpkg: dependency problems prevent configuration of foundationdb-server:
 foundationdb-server depends on foundationdb-clients (= 6.1.0); however:
  Package foundationdb-clients is not configured yet.
 foundationdb-server depends on python (>= 2.6); however:
  Package python is not installed.
 foundationdb-server depends on foundationdb-clients (= 6.1.0); however:
  Package foundationdb-clients is not configured yet.
```

Reply:
foundationdb-server depends on python (>= 2.6); however:
  Package python is not installed.

I thought I removed this?  #1015

Reply:
So you did. Then there's a further bug here that the cmake built foundationdb-server deb depends on python. I'll address that as well in this PR

Reply:
the test executes two commands: `dpkg` to install these packages (as you see above) and `apt-get -f install` to fix the dependencies. So in that case python should be installed